Content.Server/Cargo/Systems/CargoSystem.Console.cs

```diff
@@ -24,12 +24,12 @@ public sealed partial class CargoSystem
         /// <summary>
         /// How much time to wait (in seconds) before increasing bank accounts balance.
         /// </summary>
-        private const float Delay = 10f;
+        private const int Delay = 10;
 
         /// <summary>
-        /// How many points to give to every bank account every <see cref="Delay"/> seconds.
+        /// How many points to give to every bank account every second.
         /// </summary>
-        private const int PointIncrease = 10000;
+        private const int PointIncrease = 5;
```

Comment:
Shouldn't these be part of the stationbankaccountcomponent? That'd also allow them to be configured per-station in StationConfig/StationInfo (I forgot what I named it and cba to check).

Reply:
I did I just haven't pushed it.